functions/Invoke-DbaDatabaseShrink.ps1

```diff
@@ -145,24 +145,15 @@ function Invoke-DbaDatabaseShrink {
         $StatementTimeoutSeconds = $StatementTimeout * 60
 
         $sql = "SELECT
-                indexstats.avg_fragmentation_in_percent
-                FROM sys.dm_db_index_physical_stats (DB_ID(), NULL, NULL, NULL, NULL) AS indexstats
-                INNER JOIN sys.tables dbtables on dbtables.[object_id] = indexstats.[object_id]
-                INNER JOIN sys.schemas dbschemas on dbtables.[schema_id] = dbschemas.[schema_id]
-                INNER JOIN sys.indexes AS dbindexes ON dbindexes.[object_id] = indexstats.[object_id]
-                AND indexstats.index_id = dbindexes.index_id
-                WHERE indexstats.database_id = DB_ID() AND indexstats.avg_fragmentation_in_percent > 0
-                ORDER BY indexstats.avg_fragmentation_in_percent desc"
-
-        $sqlTop1 = "SELECT top 1
-                indexstats.avg_fragmentation_in_percent
-                FROM sys.dm_db_index_physical_stats (DB_ID(), NULL, NULL, NULL, NULL) AS indexstats
-                INNER JOIN sys.tables dbtables on dbtables.[object_id] = indexstats.[object_id]
-                INNER JOIN sys.schemas dbschemas on dbtables.[schema_id] = dbschemas.[schema_id]
-                INNER JOIN sys.indexes AS dbindexes ON dbindexes.[object_id] = indexstats.[object_id]
-                AND indexstats.index_id = dbindexes.index_id
-                WHERE indexstats.database_id = DB_ID()
-                ORDER BY indexstats.avg_fragmentation_in_percent desc"
+                    avg(indexstats.avg_fragmentation_in_percent) as [avg_fragmentation_in_percent],
```

Comment:
Why are we averaging a value that is already averaged?

Reply:
Fixed T-SQL =)
I get avg fragmentation in all database indexes